src/senaite/core/browser/static/js/bika.lims.analysisrequest.add.js

```diff
@@ -2376,7 +2376,8 @@
           dialog.on("yes", function() {
             // Re-submit
             $("input[name=confirmed]").val("1");
-            return $("input[name=save_button]").trigger("click");
+            me.form_submission_flag = false;
+            $("input[name=save_button]").trigger("click");
```

Comment:
This javascript is generated by compiling the `coffee` file:
https://github.com/senaite/senaite.core/blob/2.x/src/senaite/core/browser/static/js/coffee/bika.lims.analysisrequest.add.coffee

Please, do the changes in that `coffee` file and compile afterwards, as stated in the header of the file: 

https://github.com/senaite/senaite.core/blob/b620cc42636169c8834a02cf477723a45c92aa63/src/senaite/core/browser/static/js/coffee/bika.lims.analysisrequest.add.coffee#L1-L3